google-cloud-storage/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+### 1.17.0 / 2019-02-07
+
+* Add Bucket#policy_only? and #policy_only=
```

Comment:
> Add support for Bucket Policy Only with Bucket#policy_only?, #policy_only and #policy_only_locked_at, read more at https://cloud.google.com/storage/docs/bucket-policy-only.

Note: The page will be published next week so it will 404.

Reply:
```suggestion
* Add support for Bucket Policy Only with `Bucket#policy_only?`,
  `Bucket#policy_only` and `Bucket#policy_only_locked_at`.
  Read more at https://cloud.google.com/storage/docs/bucket-policy-only
```